src/device/deletePendingInvitation.js

```diff
@@ -0,0 +1,21 @@
+const { default: API } = require('@aws-amplify/api-rest')
+
+const serializeAxiosResponse = require('../utils/serializeAxiosResponse')
+const { DEFAULT_REQ_OPTS } = require('../utils/defaultReqOpts')
+
+/**
+ * @desc retract user pending invitation
+ * @memberof nexus.device
+ * @function retract
```

Comment:
the function is called `deletePendingInvitation`

the order of params should (deviceId, invitationId), so that the path parameters are standalone